Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -283,6 +302,7 @@ AttributedStringViewIssueDelegate {
         alert.addActions([
             shareAction(sender: sender),
             deleteAction(),
+            editAction(),
```

Comment:
In my head it makes more sense to have the edit action before delete? Anyone else agree?

Reply:
Hmm yeah, I feel like the delete should at least not be in the middle.

Reply:
Oh ya great call